avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastStringableTest.java

```diff
@@ -143,11 +147,24 @@ public void serializeStringableFields() throws URISyntaxException, MalformedURLE
   }
 
   @Test(groups = {"deserializationTest"}, dataProvider = "SlowFastDeserializer")
-  public void javaStringPropertyTest(Boolean whetherUseFastDeserializer) throws IOException {
-    Schema schema = createRecord(createField("testString", javaStringSchema));
+  @SuppressWarnings("unchecked")
+  public void javaStringPropertyTest(Boolean whetherUseFastDeserializer) {
```

Comment:
Could you commit  the changed files in codegen directory as well? so that we could see the differences before/after this code gen logic change.
Sample PR:
https://github.com/linkedin/avro-util/pull/70/files

Essentially, we would split the code change into  two parts:
1. core code change;
2. the generated fast serializer/de-serializer change;

With this way, we will be able to review those two set of changes separately by choosing  different commits.

For the PR link I shared, it  contains the following two commits:
"avro.java.string" support in union type for fast serializer

@gaojieliu
gaojieliu committed 13 days ago
 
Code gen for #70

@gaojieliu
gaojieliu committed 13 days ago


Let me know if you are still not clear about this new implicit convention formed inside LinkedIn.

Reply:
Hi,

Thank you for clarification, the PR has been changed according to your suggestions.